pkg/test-infra/fixture/fixture.go

```diff
@@ -157,6 +157,7 @@ func init() {
 	flag.StringVar(&Context.HistoryFile, "history", "./history.log", "history file record client operation")
 
 	flag.StringVar(&Context.Namespace, "namespace", "", "test namespace")
+	flag.StringVar(&Context.MySQLVersion, "mysql-version", "5.6", "Default mysql version")
```

Comment:
change mysql-version into `cdc.mysql-vesion`?

Reply:
At present, this flag is not used by cdc.